.travis.yml

```diff
@@ -202,36 +213,23 @@ jobs:
       distributions: sdist bdist_wheel
       on:
         repo: camptocamp/c2cgeoportal
-        tags: true
-        condition: '"${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\..+$'
-    - provider: script
-      script: ./docker-run make transifex-send
-      skip_cleanup: true
-      on:
-        repo: camptocamp/c2cgeoportal
-        branch: master
-        condition: '"${TRAVIS_PULL_REQUEST}" = false'
-    - provider: script
-      script: travis/doc.sh
-      skip_cleanup: true
-      on:
-        repo: camptocamp/c2cgeoportal
-        branch: master
-        condition: '"${TRAVIS_PULL_REQUEST}" = false'
+
+  - stage: Publish
+    script: []
+    if: NOT type = pull_request
```

Comment:
`NOT type = pull_request AND branch = master` like at the other places?

Reply:
We want to publish Docker on all the branches (as previously) :-)